CC_ggplot2_cheatsheet.Rmd

```diff
@@ -0,0 +1,169 @@
+# ggplot2_cheatsheet
+
+Xiaorui Zhang
+# 1. Some Advanced Techs from ggplot2 
```

Comment:
As per 2.2.3 (https://jtr13.github.io/cc21/github-submission-instructions.html) do not use single # anywhere other than first line of Rmd file. Please remove the same from line 4.

Reply:
ok, I got it. Thank you very much!